app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -463,6 +464,19 @@ struct DeviceDelegateOpenXR::State {
       sessionBeginInfo.primaryViewConfigurationType = viewConfigType;
       CHECK_XRCMD(xrBeginSession(session, &sessionBeginInfo));
       vrReady = true;
+
+      // If hand tracking is supported, we want to emulate a default interaction
+      // profile, so that if Wolvic is launched without controllers active, we can
+      // still use hand tracking for emulating the controllers.
+      // This is a temporary situation while we don't implement WebXR hand tracking
+      // APIs.
+      if (mHandTrackingSupported && controllersReadyCallback != nullptr && input) {
```

Comment:
Nit: you don't need the `!= nullptr`, checking for `controllersReadyCallback is enough.